display.go

```diff
@@ -1,12 +1,24 @@
 package kvm
 
 import (
+	"errors"
 	"fmt"
 	"log"
+	"os"
+	"strconv"
 	"time"
 )
 
 var currentScreen = "ui_Boot_Screen"
+var backlightState = 0 // 0 - NORMAL, 1 - DIMMED, 2 - OFF
+
+var dim_ticker *time.Ticker
+var off_ticker *time.Ticker
```

Comment:
I think these variables (and the ones above) could be merged into blocks like this:
```suggestion
var (
	dim_ticker *time.Ticker
	off_ticker *time.Ticker
)
```

Also - Effective Go and the Go stdlib usually prefers lowerCamelCase/UpperCamelCase (depending on whether exported) for names. E.g., per https://go.dev/wiki/CodeReviewComments:
> See https://go.dev/doc/effective_go#mixed-caps. This applies even when it breaks conventions in other languages. For example an unexported constant is maxLength not MaxLength or MAX_LENGTH.

Some of the function names below also contain underscores, which is not best pratice.

Reply:
Thanks, shifted those into one declaration and also renamed them to `dimTicker` and `offTicker` respectively. 

Thank you for your review!